userspace/libscap/engine/gvisor/scap_gvisor.cpp

```diff
@@ -534,8 +553,10 @@ int32_t engine::next(scap_evt **pevent, uint16_t *pdevid, uint32_t *pflags)
 		sandbox_entry &sandbox = it->second;
 		if(sandbox.m_closing)
 		{
+			std::string container_id = sandbox.m_container_id;
```

Comment:
Shouldn't this be `sandbox.m_id`? 
I mean, `release_sandbox_id` takes the sandbox id as argument...

Reply:
The names are a little confusing I understand, but the `container_id` is the string which is the one that `release_sandbox_id` takes as parameter